airflow-core/docs/core-concepts/overview.rst

```diff
@@ -244,6 +244,6 @@ User interface
 
 Airflow comes with a user interface that lets you see what Dags and their tasks are doing, trigger runs of Dags, view logs, and do some limited debugging and resolution of problems with your Dags.
 
-.. image:: ../img/ui-dark/dags.png
+.. image:: ../img/ui-light/dags.png
```

Comment:
Any reason to use light over dark?

Reply:
best practice is to follow system settings and give a choice, but where not possible we should use light mode as it works well with browsers in both light and dark, and is generally more accessible due to its higher contrast and similarity.

or so I've been told :)